experiments/oscillogram_classification.ipynb

```diff
@@ -667,7 +667,91 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "053502a3",
+   "id": "34ebd170-aaa7-4508-8311-156ff4c2b371",
```

Comment:
This way it is very tedious to recognize actual changes. Can you try to adapt it so that only your actual changes are visible? E.g., by not moving code to different cells. As far as I can tell, the only changes are to put the code blocks into functions here. It would be good if only this change would be visible. Below, this happens a couple of times.